### PR TITLE
chore: update author in Cargo.toml

### PR DESCRIPTION
**Pull request by DylanVerstraete**
# Description of proposed changes

- Removed Nathan as owner because I see he is now working for deno
- Added codeowners file (probably need to extend the blockchain engineering team members)
---

Practical tips for PR review & merge:

- [x] All GitHub Actions report PASS
- [x] Newly added code/functions have unit tests
  - [ ] Coverage tools report all newly added lines as covered
  - [ ] The positive scenario is exercised
  - [ ] Negative scenarios are exercised, e.g. assert on all possible errors
  - [ ] Assert on events triggered if applicable
  - [ ] Assert on changes made to storage if applicable
- [x] Modified behavior/functions - try to make sure above test items are covered
- [x] Integration tests are added if applicable/needed

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -4,7 +4,7 @@ resolver = "2"
 
 [workspace.package]
 version = "3.21.0"
-authors = ["Gluwa Inc.", "Nathan Whitaker <nathan.whitaker@gluwa.com>"]
+authors = ["Gluwa Inc. <info@gluwa.com>"]
```
Comment (atodorov) [inline Cargo.toml:7]: I'm not sure who reads this address. Ideally we'd like to be using an address that reaches the team directly but I'm not aware of such an address.

The rest LGTM.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Default owners for everything in the repository
+
+* @gluwa/blockchain-engineering
```
Comment (beqaabu) [inline .github/CODEOWNERS:3]: I think @gluwa/creditcoin would make more sense, we are all members there, but not sure

Comment (DylanVerstraete) [reply .github/CODEOWNERS]: Alright!

**Review by beqaabu — APPROVED** [review]

**Review by atodorov — APPROVED** [review]
LGTM

**Review by atodorov — CHANGES_REQUESTED** [review]
GitHub isn't letting me merge this. It probably wants a rebase :-(.

**Review by atodorov — APPROVED** [review]

**Pull request merged**